azure/template.json

```diff
@@ -201,6 +204,10 @@
                                 {
                                     "name": "Cdn:Url",
                                     "value": "[parameters('cdnUrl')]"
+                                },
+                                {
```

Comment:
Again I think app settings should just be in the config

Reply:
Removed this